[2025-02-17T18:45:41.849+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: etl_pipeline.extract manual__2025-02-17T18:45:41.105388+00:00 [queued]>
[2025-02-17T18:45:41.866+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: etl_pipeline.extract manual__2025-02-17T18:45:41.105388+00:00 [queued]>
[2025-02-17T18:45:41.867+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-02-17T18:45:41.869+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-02-17T18:45:41.870+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-02-17T18:45:41.891+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): extract> on 2025-02-17 18:45:41.105388+00:00
[2025-02-17T18:45:41.897+0000] {standard_task_runner.py:55} INFO - Started process 278 to run task
[2025-02-17T18:45:41.902+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'extract', 'manual__2025-02-17T18:45:41.105388+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpdi_g45c_']
[2025-02-17T18:45:41.905+0000] {standard_task_runner.py:83} INFO - Job 6: Subtask extract
[2025-02-17T18:45:41.929+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-17T18:45:41.995+0000] {task_command.py:388} INFO - Running <TaskInstance: etl_pipeline.extract manual__2025-02-17T18:45:41.105388+00:00 [running]> on host f05cc992ddf9
[2025-02-17T18:45:42.095+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_pipeline
AIRFLOW_CTX_TASK_ID=extract
AIRFLOW_CTX_EXECUTION_DATE=2025-02-17T18:45:41.105388+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-17T18:45:41.105388+00:00
[2025-02-17T18:45:42.097+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2025-02-17T18:45:42.098+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "üîç Iniciando extracci√≥n de datos..." && sleep 2 && echo "‚úÖ Extracci√≥n completada."']
[2025-02-17T18:45:42.109+0000] {subprocess.py:86} INFO - Output:
[2025-02-17T18:45:42.110+0000] {subprocess.py:93} INFO - üîç Iniciando extracci√≥n de datos...
[2025-02-17T18:45:44.113+0000] {subprocess.py:93} INFO - ‚úÖ Extracci√≥n completada.
[2025-02-17T18:45:44.115+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-02-17T18:45:44.181+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=extract, execution_date=20250217T184541, start_date=20250217T184541, end_date=20250217T184544
[2025-02-17T18:45:44.210+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-02-17T18:45:44.252+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
